news/_posts/2016-09-14-introducing-prestonbot.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "Introducing PrestonBot"
+subtitle: "The PrestaShop community best friend"
+date:   2016-09-14 11:00:00
+author: [ mandrieu ]
+icon: fa-users
+tags:
+ - contributing
+---
+
+Some of you may have noticed that since weeks, we have welcome a new team member in the core team: Preston!
```

Comment:
a weeks?
Some of you may have noticed that a few weeks ago we welcomed a new member in the core team: Preston!